core/server/models/relations/authors.js

```diff
@@ -79,6 +79,14 @@ module.exports.extendModel = function extendModel(Post, Posts, ghostBookshelf) {
 
         // NOTE: sending `post.author = {}` was always ignored [unsupported]
         onCreating: function onCreating(model, attrs, options) {
+            // there is no context.user for API Key requests so we can't
+            // auto-assign the currently logged in user
+            if (options.context && options.context.api_key_id && !model.get('author_id')) {
```

Comment:
I thought `this.contextUser(options)` returns the correct id to store for `author_id`  🤔 Let me know if i am wrong

Reply:
Nope :) That PR was closed, but also we're only auto assigning the user for `created_at` etc... fields :relaxed:

Reply:
Does that mean, if the integration adds a post, the integration needs to choose an `author_id`, right?

Reply:
Yeah that's right :+1: